services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -860,12 +863,29 @@ open class HankeServiceImpl(
      * - Hanke name is same as application name (limited to first 100 characters).
      * - Perustaja generated from application data orderer.
      */
-    private fun generateHankeFrom(cableReport: CableReportWithoutHanke): Hanke =
-        createHankeInternal(
-            CreateHankeRequest(nimi = limitHankeName(cableReport.applicationData.name)),
+    private fun generateHankeFrom(cableReport: CableReportWithoutHanke): Hanke {
```

Comment:
The comment could mention the stuff about areas. Alternatively, the comment can be removed and let the code explain itself.

Reply:
I added a line about the areas.